static/js/containers/UserPage_test.js

```diff
@@ -156,37 +177,39 @@ describe("UserPage", function() {
         UPDATE_PROFILE_VALIDATION,
       ];
 
```

Comment:
I'm a little unsure of how to go about that though, is it even possible for the user to enter invalid input in a radio field? I think we always supply a default selection.